flaskfront/opgflaskfront/templates/base.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'govuk_frontend_jinja/template.html' %}
+
+{% set assetPath = url_for('static', filename='').rstrip('/') %}
+
+{% block pageTitle %}Make a lasting power of attorney{% endblock %}
+
+{% block head %}
+  <meta name="description" content="Make a lasting power of attorney">
+  <meta name="keywords" content="">
+  <!--[if gt IE 8]><!--><link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='govuk-frontend-3.14.0.min.css') }}" /><!--<![endif]-->
+  <!--[if IE 8]><link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='govuk-frontend-ie8-3.14.0.min.css') }}" /><![endif]-->
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    'homepageUrl': url_for('home_blueprint.home'),
+    'serviceName': "Make a lasting power of attorney",
+    'serviceUrl': url_for('home_blueprint.home')
+  }) }}
+{% endblock %}
+
+{% block footer %}
+  {{ govukFooter({
```

Comment:
indentation could be clearer here, I find this a bit confusing as-is

Reply:
This is how original from land reg was.  What would you suggest to change?  I'm a little loath to change it because it'll make diffs less clear

Reply:
fair point, leave it as is